src/client/index.html

```diff
@@ -225,6 +225,14 @@
             <span data-i18n="main.join_discord"> Join the Discord! </span>
           </a>
         </div>
+        <div>
+          <button
+            id="support-us-button"
+            class="w-full bg-green-600 hover:bg-green-700 text-white py-4 px-6 text-xl rounded-lg transition-colors duration-300 flex items-center justify-center gap-4"
```

Comment:
If we want the font to be the same as the discord button:

```suggestion
            class="w-full bg-green-600 hover:bg-green-700 text-white py-4 px-6 text-lg sm:text-xl lg:text-2xl rounded-lg transition-colors duration-300 flex items-center justify-center gap-4"
```